code/src/test/test__sbi_probe_extension/test__sbi_probe_extension.c

```diff
@@ -0,0 +1,63 @@
+#include "test_macros.h"
+
+int test_case(void);
+
+/**
+ * @brief interrupt handler function that runs in S-mode. Not used in this test
+ * 
+ */
+__attribute__((interrupt("supervisor")))
+void s_mode_trap(void) {
+  uint64_t cause = read_csr(scause);
+
+  switch (cause) {
+    default:
+      break;
+  }
+}
+
+/**
+ * @brief Test case example checks for illegal instruction exception
+ * 
+ * @return int 
+ */
+int test_case(void) {
+  struct sbiret get_spec = sbi_probe_extension(EID_10);
+  if (get_spec.value == 1) {
+    if (get_spec.error == SBI_SUCCESS) {
+        exit_test(TEST_PASS);}
+  } else {
+    exit_test(TEST_FAIL);}
+
+/**
```

Comment:
Move the description below to the correct location above the test case function, replacing it.